### PR TITLE
Issue71 Changes Add to Master

### PR DESCRIPTION
**Pull request by jwbowers**
Since the Issue71 fix from @benthestatistician worked well, we should probably include that in the master branch. I also had to fix some spelling and change http to https to satisfy the latest CRAN tools. And probably we should change our cran-comments.md to the issue71 version. I'm making this pull request to start the conversation about how to pull those changes into the master branch (and, I hope, then delete the issue71 branch).

### CODE REVIEW

**Comment by benthestatistician** [comment]
My fix for the #93 problem, 544332e, is already in the master branch.  The issue71 branch is sort of a stub of our work as it stood about 3 years ago now, and I don't think we should merge it into master yet.  

OTOH I also don't think we should delete it yet: although I'm optimistic that the master branch will be ready for release before the next CRAN missive instructing us to take care of this or that warning, I'd hate to create a situation where other priorities can get held hostage to a CRAN fixit deadline. 

In summary I'd argue we shouldn't do anything more here.  Accordingly I'm closing the merge request.  Disagree-ers should feel free to re-open.
